allofplos/tests/test_partial_dois.py

```diff
@@ -0,0 +1,52 @@
+from .. import Corpus, Article, starterdir
+from ..plos_regex import validate_partial_doi, validate_doi
+from ..transformations import partial_to_doi, doi_to_partial
+
+import pytest
+
+
+@pytest.fixture
+def corpus():
+    return Corpus(starterdir, seed=1000)
+
+
+@pytest.fixture
+def test_article():
+    return Article('10.1371/journal.pone.0040259', directory=starterdir)
+
+
+@pytest.fixture
+def test_doi():
```

Comment:
Same as with `test_article` shouldn't be prefixed with test, better is to just use `def doi():`